.azure-pipelines/cleanup.yml

```diff
@@ -10,8 +10,5 @@ steps:
     if sudo [ -f dockerfs/var/run/docker.pid ] ; then
         pid=`sudo cat dockerfs/var/run/docker.pid` ; sudo kill $pid
     fi
-    sudo rm -rf dockerfs
-    sudo rm -rf fsroot
-    username=$(id -un)
-    sudo chown -R ${username}.${username} .
+    sudo rm -rf $(ls -A1)
```

Comment:
is this neccessary? if we delete everything, then we will having some difficult time to debug anything.

Reply:
neccessary. Without this PR build maybe only need 5min. Because target is already exist.

Reply:
we can remove clean at the end.